website/demos/TreeView.tsx

```diff
@@ -120,7 +120,8 @@ const defaultRows = createRows();
 export default function TreeView({ direction }: Props) {
   const [rows, dispatch] = useReducer(reducer, defaultRows);
   const [allowDelete, setAllowDelete] = useState(true);
-  const columns: Column<Row>[] = useMemo(() => {
+
+  const columns = useMemo((): readonly Column<Row>[] => {
```

Comment:
FYI, because of where the type was set, doing the global rename did not automatically rename `formatter` here.
Another reason to always define the type as close as possible to the actual values.